fix(minifier): do not compare bigint with object

7086cmd opened the pull request:
@Boshen, could you please update the snap of runtime and commit to this PR? I want to see the effects after the unit tests are added.

7086cmd commented:
* **#7294** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7294?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7294?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Boshen commented on crates/oxc_minifier/src/ast_passes/peephole_fold_constants.rs:
```diff
@@ -873,6 +873,14 @@ mod test {
         test("'1' !== 1n", "true");
     }
 
+    // Related: https://github.com/oxc-project/oxc/blob/8d8f34cfb5b1898959456b86eda9c1703bf31d99/tasks/coverage/snapshots/runtime.snap#L451
```
```suggestion
```

Boshen commented:
### Merge activity

* **Nov 16, 12:40 AM EST**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

7086cmd committed:
fix(minifier): do not compare bigint with object (#7294)

@Boshen, could you please update the snap of runtime and commit to this PR? I want to see the effects after the unit tests are added.
